content/master/operations/watchoperation.md

```diff
@@ -0,0 +1,600 @@
+kubectl get operations -l crossplane.io/watchoperation=my-watchop
+
+# Check watched resource count
+kubectl describe watchoperation my-watchop
+
+# Check events
+kubectl get events --field-selector involvedObject.name=my-watchop
+```
+
+
+## Best practices
+
+### Resource selection
+
+1. **Use specific label selectors** - Prevent unnecessary Operations with 
+   precise filtering
+1. **Avoid high-churn resources** - Be careful watching frequently changing 
+   resources
+1. **Start small** - Begin with narrow selectors and expand as needed
+
+### Event handling
+
+1. **Implement event filtering** - Check generation, deletion timestamp, 
+   and status conditions
+   to avoid processing irrelevant changes
+<!-- vale write-good.TooWordy = NO -->
+1. **Monitor operation volume** - Popular resources can create numerous 
+   Operations
+<!-- vale write-good.TooWordy = YES -->
+
+### Concurrency policies
+
+1. **Choose appropriate concurrency policies**:
+   - **Allow** for independent processing that can run in parallel
+   - **Forbid** for operations that must complete before processing new 
+     changes  
+   - **Replace** for status-checking or monitoring where only latest state 
+     matters
+
+### History management
+
+Like CronOperations, WatchOperations automatically clean up completed Operations:
+
+```yaml
+apiVersion: ops.crossplane.io/v1alpha1
+kind: WatchOperation
+metadata:
+  name: config-validator
+spec:
+  watch:
+    apiVersion: v1
+    kind: ConfigMap
+  successfulHistoryLimit: 10  # Keep 10 successful Operations (default: 3)
+  failedHistoryLimit: 5       # Keep 5 failed Operations (default: 1)
+  operationTemplate:
+    # Operation template here
+```
+
+### Watched resource injection
+
+<!-- vale write-good.TooWordy = NO -->
+WatchOperations automatically inject the changed resource into the created 
+Operation using a special requirement name 
+`ops.crossplane.io/watched-resource`:
+<!-- vale write-good.TooWordy = YES -->
+
+```python
+from crossplane.function import request, response
+
+def operate(req, rsp):
+    # Access the resource that triggered this Operation
+    watched_resource = request.get_required_resource(req, "ops.crossplane.io/watched-resource")
+    if not watched_resource:
+        response.set_output(rsp, {"error": "No watched resource found"})
+        return
+    
+    # Process based on the watched resource
+    if watched_resource["kind"] == "ConfigMap":
+        config_data = watched_resource["data"]
+        # Validate configuration...
+```
+
+The watched resource is available in the function's `required_resources` map 
+without needing to declare it in the Operation template.
+
+For general Operations best practices including function development and 
+operational considerations, see [Operation best practices]({{<ref "operation#best-practices">}}).
+
+## Troubleshooting
+
+<!-- vale Google.Headings = NO -->
+### WatchOperation not creating Operations
```

Comment:
worth adding an item in the troubleshooting section to ensure the `--enable-operations` flag has been passed to enable the feature?